Tests/misc/data/xmlSubFile.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ttFont ttLibVersion="3.15">
+
+  <TTGlyph name=".notdef" xMin="65" yMin="-60" xMax="495" yMax="770">
+    <contour>
+      <pt x="65" y="770" on="1"/>
+      <pt x="95" y="770" on="1"/>
+      <pt x="95" y="-60" on="1"/>
+      <pt x="65" y="-60" on="1"/>
+    </contour>
+   <contour.../>
```

Comment:
looks like this test file "xmlSubFile.xml" is not used. if so, they we should remove it